resources/assets/styles/blocks/menu.scss

```diff
@@ -0,0 +1,41 @@
+.menu-iq-menu-container {
+  width: 100%;
+}
+
+.menu {
+  list-style: none;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+  margin: 0 auto;
+  height: 100%;
+
```

Comment:
Czy na pewno tu będzie te 90%, i nie widzę żadnego backgroundu, obawiam się że gdy pojawi się content pod spodem to pojawią się z tym problemu